### PR TITLE
Front end i18n updates

### LINKED ISSUE #1000: Messages hard-coded in english in js
Hi some message are hard-coded in english, so they are not translated (in french for me).
2017-11-27 23_28_30-selfoss 169

Examples of hard-coded english messages in javascript file (with code source):

-var refreshed=**'Sources have been refreshed'**...
-selfoss.ui.showMessage(refreshed,**'Reload list'**...
-selfoss.ui.showError(**'Cannot refresh sources: '**...

Possible french translation:
- 'Les sources ont été rafraichies'
- 'Recharger la liste'
- 'Impossible de rafraichir les sources : ' (in french there is a white space before colon ':')

### PR DESCRIPTION
**Pull request by Seanmcn**
Resolves #1000 adding in the suggested translation in French, the rest with Google Translate..

Adds variables in a number of other places on the front-end where there is hardcoded English. 

Happy #hacktoberfest 😄

### CODE REVIEW

**Comment by jtojnar** [comment]
Thank you very much for your contribution.

1. Could you make it so that the error messages’ ids start with `error_`?
2. Adding the commented out translations might confuse the translation website (coming soon). Could you remove the commit?

----

@Binnette do you think you could check the French translations?

**Comment by Seanmcn** [comment]
Have made the requested changes, let me know if you need anything else from me on this. 😄

**Comment by jtojnar** [comment]
I think the colons should go to the translations too – some languages might use different punctuation.

File: public/js/selfoss-events.js [hunk 1]
```diff
@@ -174,7 +174,7 @@ selfoss.events = {
                         selfoss.filter.sourcesNav = true;
                     }
                 } else if (selfoss.events.subsection != 'all') {
-                    selfoss.ui.showError('Invalid subsection: '
+                    selfoss.ui.showError(F + ': '
```
Comment (jtojnar) [inline public/js/selfoss-events.js]: What is this?

Comment (Seanmcn) [reply public/js/selfoss-events.js]: Hotkey fail I didn't notice until reviewing it in Github. 😅

File: public/lang/fr.ini [hunk 2]
```diff
@@ -53,3 +53,19 @@ lang_login_invalid_credentials=Nom d’utilisateur/Mot de passe incorrects
...
+lang_error_unknown_tag=Étiquette inconnue
+lang_error_unknown_source=Identifiant de source inconnu
+lang_error_mark_items=Impossible de marquer tous les éléments visibles
+lang_error_logout=Impossible de se déconnecter
+lang_error_loading=Erreur de chargement
+lang_error_loading_stats=Impossible de charger les statistiques de navigation
+lang_error_saving_color=Impossible d'enregistrer une nouvelle couleur
+lang_error_refreshing_source=Impossible de rafraichir les sources
+lang_error_sync=Impossible de synchroniser les dernières modifications du serveur
+lang_error_invalid_subsection=Sous-section invalide
```
Comment (jtojnar) [inline public/lang/fr.ini]: Could you add the final line break? On UNIX systems the lines are terminated by it.

Comment (Seanmcn) [reply public/lang/fr.ini]: Done 😄

**Comment by Binnette** [comment]
Done ! I just reviewed french string and I made a pull request in @Seanmcn repo : Seanmcn/selfoss#1

2 strings were missing in fr.ini : 
- lang_error_mark_item
- lang_error_star_item

Can you please chek that those 2 string are not missing in other files ?
See you 😉

**Comment by jtojnar** [comment]
Thank you both.

**Comment by jtojnar** [comment]
Hi @Seanmcn and @Binnette. Previously, it [was not clear](https://github.com/fossar/selfoss/issues/1218) if selfoss was licensed under GPL 3 only, or also any later version. Could you clarify whether you are fine with licensing your contribution under `GPL-3-or-later`?

Thanks again and sorry for the confusion.

**Comment by Seanmcn** [comment]
@jtojnar fine with me 👍

**Comment by Binnette** [comment]
Of course @jtojnar 👍 😉

**Pull request merged**